app/Helpers/database/forum.php

```diff
@@ -3,7 +3,9 @@
 use App\Community\Enums\ArticleType;
 use App\Community\Enums\SubscriptionSubjectType;
 use App\Enums\Permissions;
+use App\Models\ForumTopic;
 use App\Models\Game;
+use App\Models\User;
```

Comment:
```
Unknown column 'Author' in 'field list'
```
error on line 182 trying to create a new forum topic

Reply:
Resolved in latest.